hw/xbox/dsp/dsp_dma.c

```diff
@@ -221,6 +221,7 @@ static void dsp_dma_run(DSPDMAState *s)
                  scratch_size);
 
 
+        unsigned int transfer_size = count * item_size;
```

Comment:
nit: Prefer `const size_t`

Reply:
I agree, and will consider changing it to `size_t`. Reason against it might still be what also affected https://github.com/xqemu/xqemu/pull/167#discussion_r244278041.

I don't agree regarding `const`.

Reply:
I changed it to `size_t` but not a `const` for the reasons I've mentioned here.